web/services/file.service.ts

```diff
@@ -61,6 +63,14 @@ export class FileService extends APIService {
     this.cancelSource.cancel("Upload cancelled");
   }
 
+  // async validateFile(assetUrlWithWorkspaceId: string): Promise<any> {
```

Comment:
please remove unused comments

Reply:
Done in https://github.com/makeplane/plane/pull/2789